ebuild-writing/common-mistakes/text.xml

```diff
@@ -265,7 +275,7 @@ part.
     For example, it should be <c>&gt;=x11-libs/gtk+-2</c> rather than
     <c>&gt;=x11-libs/gtk+-2*</c>.
   </li>
-  <li><e>GTK specific. Always use =x11-libs/gtk+-1.2* for GTK+1 apps.</e></li>
+  <li><e>GTK specific. Always use <c>=x11-libs/gtk+-1.2*</c> for GTK+-1 apps.</e></li>
```

Comment:
I think this bullet point should just be dropped. I don't understand what it's trying to say, and GTK+-1 is long gone.

Reply:
Yeah, I spent too long trying to think of a contemporary example and failed. It still doesn't add anything, so let's remove it.